desktop/libs/notebook/src/notebook/connectors/trino.py

```diff
@@ -90,21 +87,24 @@ def get_auth_password(self):
         else DEFAULT_AUTH_PASSWORD.get()
     )
 
+  def _format_database(self, database):
+    parts = database.split('.')
+    database = '"' + '"."'.join(parts) + '"'
```

Comment:
Shouldn't we make sure only formats it once? I.e. not if it's already formatted..

Reply:
Done